ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -108,43 +128,26 @@
    *                 listen to {@link Account} change messages. Can be {@code null}.
    * @param scheduler A {@link ScheduledExecutorService} that will run thread to update accounts in background.
    *                  {@code null} to disable background updating.
-   * @param storeConfig The configs for {@code HelixAccountService}.
+   * @param config The configs for {@code HelixAccountService}.
+   * @throws IOException
    */
   HelixAccountService(HelixPropertyStore<ZNRecord> helixStore, AccountServiceMetrics accountServiceMetrics,
-      Notifier<String> notifier, ScheduledExecutorService scheduler, HelixPropertyStoreConfig storeConfig) {
+      Notifier<String> notifier, ScheduledExecutorService scheduler, HelixAccountServiceConfig config)
+      throws IOException {
```

Comment:
Where does the IOException come from?  Can you include that in the description?

Reply:
It happens if the backup dir does not exist and cannot be created. I can add that to the description.